codeHF/config_tasks.sh

```diff
@@ -141,6 +149,9 @@ function AdjustJson {
 # Generate the O2 script containing the full workflow specification.
 function MakeScriptO2 {
   WORKFLOWS=""
+  # Cascade reconstruction
+  [ $DOO2_CASC -eq 1 ] && SUFFIX_CASC="-v0" || SUFFIX_CASC=""  # Event selection
+  [ $DOO2_EVSEL -eq 1 ] && { WORKFLOWS+=" o2-analysis-timestamp o2-analysis-event-selection"; SUFFIX_EVSEL="-evsel"; } || SUFFIX_EVSEL=""
```

Comment:
Fix the position of `# Event selection`.

Reply:
Done